airbyte-integrations/connectors/source-surveymonkey/source_surveymonkey/schemas/survey_questions.json

```diff
@@ -21,16 +21,16 @@
       "type": "string"
     },
     "layout": {
-      "type": "string"
+      "type": ["object", "null"]
     },
     "sorting": {
-      "type": "string"
+      "type": ["string", "null"]
     },
     "required": {
-      "type": "string"
+      "type": ["string", "null"]
     },
     "validation": {
-      "type": "string"
+      "type": ["string", "null"]
     },
     "forced_ranking": {
       "type": "boolean"
```

Comment:
You admit that 4 properties can be `Null`, are you sure that the rest of the properties cannot be `Null `too?

Reply:
Rest of the properties  are required for request (or they have default value) hence they would always be present on output

 https://developer.surveymonkey.com/api/v3/#surveys-id-pages-id-questions